src/swagger.json

```diff
@@ -1,12 +1,10 @@
 {
```

Comment:
I was going to change this myself, but there's an issue with the 401 responses. They should return `{ success: false,  data: {errors: [string]} }`. I think this is p much wrong in a lot of places.